src/components/AllJobLogsView/AllJobLogsView.js

```diff
@@ -36,13 +39,25 @@ import {
 } from './CustomQueryBuilder';
 
 const excludedSortColumns = ['fileName'];
+const jobStatusFailString = 'status=(FAIL or COMPLETED_WITH_ERRORS)'
 
 const buildJobsQuery = makeQueryBuilder(
   `status=(${JOB_LOGS_STATUS_QUERY_VALUE})`,
   (query) => `query=${query}`,
   null,
   {
     completedDate: buildDateTimeRangeQuery.bind(null, ['completedDate']),
+    status: (query)=>{
+      if(query === JOB_EXECUTION_STATUSES.FAIL){
+          return jobStatusFailString
+      } else if (Array.isArray(query)) {
+        return `status=(${query.map(v => {
+          if (v === JOB_EXECUTION_STATUSES.COMPLETED) {
+            return JOB_LOGS_STATUS_QUERY_VALUE;
+          } else return `"${v}"`;
+        }).join(' or ')})`;
+      } else return `status=${query}`;
+    }
```

Comment:
```suggestion
      switch (true) {
      case query === JOB_EXECUTION_STATUSES.FAIL:
          return jobStatusFailString;
      case Array.isArray(query):
        return `status=(${query.map(v => (
          v === JOB_EXECUTION_STATUSES.COMPLETED
            ? JOB_LOGS_STATUS_QUERY_VALUE
            : `"${v}"`;
        )).join(' or ')})`;
      default: return `status=${query}`;
    }
```

Reply:
It looks better, I made that changes. Thanks @uladislausamets